src/components/data_search/DatasetFilterList.jsx

```diff
@@ -42,6 +45,38 @@ export const FilterItemList = (props) => {
   );
 };
 
+export const AccessManagementFilterItemList = (props) => {
+  const { category, datasets, filter, filterHandler, isFiltered, filterNameFn } = props;
+  return (
+    <List sx={{ margin: '-0.5em -0.5em' }}>
+      {
+        filter.map((filter) => {
+          const filterName = filterNameFn(filter);
+          return (
+            <ListItem disablePadding key={filter}>
+              <ListItemButton sx={{ padding: '0' }} onClick={(event) => filterHandler(event, datasets, category, filter)}>
+                <ListItemIcon>
+                  <Checkbox checked={isFiltered(filter, category)} />
+                </ListItemIcon>
+                <ListItemText sx={{ fontFamily: 'Montserrat', transform: 'scale(1.2)' }}>
+                  { filterName === 'Open' ? <div style={{display: 'flex', alignItems: 'center'}}>
+                    <img src={openAccessIcon} style={{width: '10px', marginRight: 6}} /> {filterName} </div> :
```

Comment:
Instead of using a nested ternary op, can this be done using a switch statement? Since all the other attributes are the same, the only thing that would change is the `src` attribute. The default case would either have to map to a placeholder icon (maybe a question mark since it's unexpected) or the external link icon.

Reply:
The images could use alt tags here.